src/constructs/loadbalancing/alb/application-load-balancer.ts

```diff
@@ -40,16 +52,32 @@ interface GuApplicationLoadBalancerProps extends ApplicationLoadBalancerProps, A
  */
 export class GuApplicationLoadBalancer extends GuAppAwareConstruct(ApplicationLoadBalancer) {
   constructor(scope: GuStack, id: string, props: GuApplicationLoadBalancerProps) {
+    const { app, removeType, withAccessLogging = true } = props;
+    const { stack, stage } = scope;
+
     super(scope, id, { deletionProtection: true, ...props });
 
     this.setAttribute(TLS_VERSION_AND_CIPHER_SUITE_HEADERS_ENABLED, "true");
     this.setAttribute(DROP_INVALID_HEADER_FIELDS_ENABLED, "true");
 
-    if (props.removeType) {
+    if (removeType) {
       const cfnLb = this.node.defaultChild as CfnLoadBalancer;
       cfnLb.addPropertyDeletionOverride("Type");
     }
 
+    if (withAccessLogging) {
+      const bucket = Bucket.fromBucketName(
+        scope,
+        AppIdentity.suffixText(props, "AccessLoggingBucket"),
+        GuAccessLoggingBucketParameter.getInstance(scope).valueAsString,
+      );
+      const prefix = `application-load-balancer/${stage}/${stack}/${app}`;
+      this.logAccessLogs(bucket, prefix);
+    } else {
+      this.setAttribute("access_logs.s3.enabled", "false");
```

Comment:
From the [AWS docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/TemplateReference/aws-properties-elasticloadbalancingv2-loadbalancer-loadbalancerattribute.html) `access_logs.s3.enabled` is implicitly `false`. Setting the value explicitly is a lot easier to reason about (and test!).